week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/types/item.ts

```diff
@@ -0,0 +1,11 @@
+export interface Price {
+  value: number;
+  currency: 'PLN' | 'USD' | 'EUR'; // TODO check required currencies
+}
+
+export interface Item {
```

Comment:
Ja proponuje Ci to nazwać jako `models/` zamiast `types/` a tak to jest super.

Model będzie podbijał ważność kształtu tych danych pod względem samej "logiki działania koszyka". 
To takie pokazanie faktu "zobaczcie operujemy na `Items`".